managed_repository/main.tf

```diff
@@ -45,41 +45,20 @@ resource "github_team" "managed_repository-internal_admins" {
   privacy     = "closed"
 }
 
-resource "github_team" "managed_repository-external_admins" {
-  count       = "${length(var.push_teams_external)}"
-  name        = "${var.project_name}-external_admins"
-  description = "${var.project_name} external admin team"
-  privacy     = "closed"
-}
-
 resource "github_team" "managed_repository-internal_pull" {
   count       = "${length(var.pull_teams_local)}"
```

Comment:
this line should be deleted too! You can't use `count` when creating one team. If the list in `${length(var.pull_teams_local)}` is longer than 1 item, it will break on the github side and if the list is 0, it won't create the team which means the code is useless.